crates/bws/src/main.rs

```diff
@@ -67,6 +68,21 @@ enum Commands {
         #[command(subcommand)]
         cmd: GetCommand,
     },
+    #[command(long_about = "Create a single item")]
+    Create {
+        #[command(subcommand)]
+        cmd: CreateCommand,
+    },
+    #[command(long_about = "Edit a single item")]
+    Edit {
+        #[command(subcommand)]
+        cmd: EditCommand,
+    },
+    #[command(long_about = "Delete one or more items")]
+    Delete {
+        #[command(subcommand)]
+        cmd: DeleteCommand,
+    },
```

Comment:
Starting to dislike the grouping of commands. Feels like we should group by type and not action, but that's outside the scope of this feature.